src/wccp2.cc

```diff
@@ -16,6 +16,7 @@
 #include "comm.h"
 #include "comm/Connection.h"
 #include "comm/Loops.h"
+#include "compat/socket.h"
```

Comment:
There is a `netdb.h` include below, it seems this file is missing `compat/netdb.h` and maybe some API changes to use it?

Reply:
neither is needed in reality. Removed both